scripts/Imaging/imgpy/filters/crop_coords.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import (absolute_import, division, print_function)
+import numpy as np
+import helper as h
+
+
+def cli_register(parser):
+    parser.add_argument(
+        "-R",
+        "--region-of-interest",
+        nargs='*',
+        required=False,
+        type=str,
+        help="Crop original images using these coordinates, after rotating the images.\n"
+        "If not given, the whole images are used.\n"
+        "Example: --region-of-interest='[150,234,23,22]'.")
+
+    return parser
+
+
+def gui_register(par):
+    raise NotImplementedError("GUI doesn't exist yet")
+
+
+def execute(sample, region_of_interest, flat, dark):
+    """
+    Execute the Circular Mask filter.
+
+    :param data: The sample image data as a 3D numpy.ndarray
+    :param region_of_interest: The region of interest that will be cropped
+    :param flat: The average flat image to be cropped
+    :param dark: The average dark image to be cropped
+
+    :return: the data after being processed with the filter
+    """
+    if sample is not None and flat is None and dark is not None:
```

Comment:
Problem here is with numpy arrays:
```
import numpy as np
ndarray_ones = np.ones((10,10,10))
if ndarray_ones:  # doesn't work, requires .any() or .all()
    print("do thing")
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```
and this might be more expensive that just a `is not None` check. In any case this is a small bug, it shouldn't check for `dark is not None`, but for `dark is None`